config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
-end
+
+  get "/mechanics/:id", to: "mechanics#show"
+  post "/mechanics/:id", to: "mechanic_rides#create"
```

Comment:
Great controller organization! A slightly more RESTful path here though would be "mechanics/:mechanic_id/rides" to indicate the relationship you're building between a given ride and mechanic (Rather than a path that just has mechanics in it)